common/metrics/otl.go

```diff
@@ -0,0 +1,77 @@
+package metrics
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+
+	"github.com/ceramicnetwork/go-cas/models"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp"
+	"go.opentelemetry.io/otel/exporters/stdout/stdoutmetric"
+	"go.opentelemetry.io/otel/metric"
+	sdk "go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/resource"
+)
+
+type OTLMetricService struct {
```

Comment:
minor: Should this be named `OtlMetricService`?

Reply:
i think not in case we change the internals later

Reply:
I'm not sure I understand. Was just recommending changing `OTL` -> `Otl` to preserve camel casing. Should we not?